transcom/mymove
# B-19975-INT Customer PPM Feedback

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19975)

## Summary

Once customers submit their PPM documents, they are unable to view details about what they submitted, as well as feedback from closeout counselors. To fix this, we added a Feedback review component - once it reaches "APPROVED" status, customers can now see details about their PPM documents, along with feedback from their closeout counselor.

Most back end changes largely were handled in previous PRs against 19988 and 19982. For the front end, at a high level: PPMSummaryList handles the logic on displaying the Feedback page. Feedback component wraps the individual feedback items and structures/formats the data. And FeedbackItems component displays the data.

### How to test

1. As a customer with an approved PPM, create trip, set, and expense documents. Take a screen shot or notate the values. Submit.
2. As a closeout counselor, view the move and review the documents. Alter different amounts of stuff and approve/exclude (do not reject, as this will send the shipment back to the PPM Document submission stage) and confirm review is complete when you reach the end.
3. As a customer, view your move and confirm you can now see a "View Closeout Feedback" button. Click that and find yourself at the Feedback review page. Compare the values/adjusted values to the values you input. Check that it works as AC requests, that it accurately reflects the white paper, and try and break it.
